docs/dev/api/api-testing.md

```diff
@@ -0,0 +1,2271 @@
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "payload",
+                    "success": true
+                }, {
+                    "action": "assert-is",
+                    "expression": "payload.message is array",
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "_1",
+                    "root": "payload.message",
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "_1",
+                    "root": "payload.message",
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "_1",
+                    "root": "payload.message",
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "_1",
+                    "root": "payload.message",
+                    "success": true
+                }, {
+                    "action": "assert-exists",
+                    "expression": "payload.status",
+                    "success": true
+                }],
+                "kind": "sequence"
+            }],
+            "kind": "inputSet",
+            "name": "default"
+        }],
+        "inputBatteryName": "default",
+        "kind": "inputBattery"
+    }],
+    "tags": ["dogs", "retrievers"],
+    "criticalFailures": [],
+    "httpFailures": [],
+    "facts": {},
+    "date": 1645952352117,
+    "test": {
+        "name": "Get all retriever sub-breeds",
+        "id": "621b20e8fd17a5416b299e9f"
+    },
+    "failuresCount": 0,
+    "warningsCount": 0,
+    "compressed": false,
+    "run": {
+        "name": "",
+        "id": ""
+    },
+    "company": {
+        "name": "",
+        "id": "7fb25570b4064716b9b6daae2a846890"
+    },
+    "project": {
+        "name": "Dog CEO",
+        "id": "621ad1466b1fa36aa4b8b044"
+    },
+    "contextId": "6cb1d39a-0964-3dfb-b595-7eabb3db1840",
+    "temp": false,
+    "expireAt": null,
+    "executionTimeSeconds": 1,
+    "taskId": "35bc67c7-c8ec-4686-b30d-47cc48e094hf",
+    "agent": "wstestjs",
+    "mode": "ondemand",
+    "buildId": "",
+    "exception": ""
+}]
+```
+
+</details>
+
+
+---
+
+### Execute a Test Synchronously
+
+<details><summary><span className="api post">POST</span><code>/api-testing/rest/v4/&#123;hookId&#125;/tests/_exec-sync</code></summary>
+<p/>
+
+Executes a test synchronously that you send in the request body. It will create a Log in the project **Dashboard**, however, the test itself will not populate in your project's **Tests** section.
+
+With this method, the API waits until all results are available, before showing them. With the [Execute a Test](#execute-a-test) method, it will just show that the execution has been done.
```

Comment:
@kimsauce this is not strictly related to just "/_exec-sync" but to all endpoints with "sync" Could you please move this statement at the top of the section " Test Execution (synchronous)" (line 831)